_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -4798,7 +4798,7 @@ mfxStatus MfxHwH264Encode::CheckVideoParamMvcQueryLike(MfxVideoParam & par)
         {
             mfxF64 rawDataBitrate = 12.0 * par.mfx.FrameInfo.Width * par.mfx.FrameInfo.Height *
                 par.mfx.FrameInfo.FrameRateExtN / par.mfx.FrameInfo.FrameRateExtD;
-            mfxU32 minTargetKbps = mfxU32(MFX_MIN(0xffffffff, rawDataBitrate / 1000 / 500));
+            mfxU32 minTargetKbps = std::min<mfxU32>(0xffffffff, rawDataBitrate / 1000.0 / 500.0);
```

Comment:
Right part can be saturated. Compare as double.

Reply:
ok